emgcli/settings.py

```diff
@@ -358,14 +358,14 @@ def create_secret_key(var_dir):
     ),
 
     'DEFAULT_RENDERER_CLASSES': (
-        'rest_framework_json_api.renderers.JSONRenderer',
+        # 'rest_framework_json_api.renderers.JSONRenderer',
         'emgapi.renderers.DefaultJSONRenderer',
```

Comment:
Can you please remove the commented renderers? I've meaning to do that for a long time